docker/entrypoint.sh

```diff
@@ -161,7 +161,7 @@ run_tests() {
       --package zebra-state \
       -- --nocapture --include-ignored with_fake_activation_heights
 
-  elif [[ "${TEST_SCAN_TASK_COMMANDS}" -eq "1" ]]; then
+  elif [[ "${TEST_SCANNER}" -eq "1" ]]; then
```

Comment:
We forgot to match the var name in CI: https://github.com/zcashfoundation/zebra/blob/57317ecb8e834ec32799ab41bb9cb5223a32015e/.github/workflows/sub-ci-integration-tests-gcp.yml#L500.